### ISSUE TITLE
Issue during overcloud deployment on Alias lab 740xd machines

### ISSUE THREAD

**Issue by sanjaychari** [redhat-performance/jetpack#409]
While running ansible-playbook -t overcloud main.yml after deploying undercloud, introspection, and tagging;
the following error occurs at the end of overcloud deployment on 740xd machine type in Alias lab.
```
TASK [Refresh the overcloud node facts] ****************************************
task path: /home/sanjay/.infrared/plugins/tripleo-overcloud/update_inventory.yml:24
fatal: [undercloud]: FAILED! => {
    "msg": "'dict object' has no attribute 'shade'"
}
```
Replacing `delegate_to: "{{ groups.shade | first }}"` with `delegate_to: undercloud` in `/home/sanjay/.infrared/plugins/tripleo-overcloud/update_inventory.yml` and `/home/sanjay/.infrared/plugins/tripleo-overcloud/clouds.yml` fixed this issue.

**Comment by sanjaychari** [comment]
This appears to happen only when running `ansible-playbook -t overcloud main.yml`. Running `ansible-playbook main.yml` does not show this issue. I think we need to include a step in our playbook to replace `delegate_to: "{{ groups.shade | first }}"` with `delegate_to: undercloud` in the 2 infrared files, using the `sed` command.

**Comment by sanjaychari** [comment]
Closed by #413
